playbooks/roles/bats/defaults/main.yaml

```diff
@@ -4,6 +4,6 @@ bats_forklift_dir: "/root/forklift"
 bats_output_dir: "/root/bats_results"
 bats_update_forklift: "yes"
 bats_tests:
-  - "fb-install-katello.bats"
```

Comment:
If we could either keep this here, or split those off as part of this then I'd be :+1: for merging